Cargo.toml

```diff
@@ -45,3 +45,9 @@ expect-test = "1.4"
 [features]
 serde = ["dep:serde", "revm/serde"]
 js-tracer = ["dep:boa_engine", "dep:boa_gc"]
+
+[patch.crates-io]
+revm = { git = "https://github.com/bluealloy/revm", branch = "inner_evm_context" }
+revm-precompile = { git = "https://github.com/bluealloy/revm", branch = "inner_evm_context" }
+revm-primitives = { git = "https://github.com/bluealloy/revm", branch = "inner_evm_context" }
+revm-interpreter = { git = "https://github.com/bluealloy/revm", branch = "inner_evm_context" }
```

Comment:
vote to just use the crates with the `branch` tag instead of the patch as it makes lib usage rly hard

Reply:
hence the draft, will be updated after merge+release

Reply:
This is just for a test, will publish revm before merging this